project/BuildDependencies/scripts/0_package.list

```diff
@@ -5,28 +5,48 @@
 ; -> sqlite-3.7.12.1-win32\project\BuildDependencies\lib\sqlite3.lib
 ; -> sqlite-3.7.12.1-win32\system\sqlite3.dll
 ; -> ...
+;PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER!
 boost-1.46.1-headers-win32.7z
+bzip2-1.0.5-win32.7z
 dnssd-541-win32.zip
 doxygen-1.8.2-win32.7z
+fontconfig-2.8.0-win32.7z
+freetype-2.4.6-win32-3.7z
+giflib-5.0.5p-win32.7z
 gnutls-3.2.3-win32.zip
 jsonschemabuilder-1.0.0-win32-3.7z
 libass-0.10.2-win32.7z
 libbluray-0.4.0-win32.zip
+libcdio-0.83-win32-2.7z
 libcec-2.2.0-win32-1.7z
+libcurl-7.21.6-win32.7z
+libexpat_2.0.1-win32.7z
+libflac-1.2.1-win32.7z
+libfribidi-0.19.2-win32.7z
 libiconv-1.14-win32-vc120.7z
 libjpeg-turbo-1.2.0-win32.7z
+liblzo-2.04-win32.7z
+libmicrohttpd-0.4.5-win32-1.7z
 libnfs-1.6.2-win32.7z
+libogg-vc100-1.2.0-win32.7z
+libplist-1.7-win32-3.7z
+libpng-1.5.13-win32.7z
+librtmp-20110723-git-b627335-win32-1.7z
```

Comment:
you know that this is the wrong file, don't you? (-1 at end) nevertheless does not matter, I will update with new lib anyway.